software/teachee-desktop/src/usb_manager.rs

```diff
@@ -0,0 +1,46 @@
+use std::{error::Error, thread, time::Duration};
+
+use libftd2xx::{BitMode, DeviceTypeError, Ft232h, FtdiCommon};
+
+use crate::storage::Storage;
+
+pub fn ft_manger(storage: &mut Storage) -> Result<(), DeviceTypeError> {
+    // See: https://ftdichip.com/wp-content/uploads/2020/08/AN_130_FT2232H_Used_In_FT245-Synchronous-FIFO-Mode.pdf
+    const MASK: u8 = 0xFF;
+    const LATENCY_TIMER: Duration = Duration::from_millis(16);
+    const IN_TRANSFER_SIZE: u32 = 0x10000;
+
+    let mut ft = Ft232h::with_description("TeachEE")?;
```

Comment:
Let's lock this in, I will flash all the boards to have this description string on the FTDI

Reply:
Sounds good.